Add CreatorProfile & UpdaterProfile explanation in orderitems.md

BrittaBecker opened the pull request:
Some endpoints include now "Creator Profile" and "Updater Profile" in the response. This is already displayed in the code example in the documentation, but not explained. I noticed it also in bills/getAll. Can we please add an explanation to all endpoints that return those values?

#### Summary

#### Follow style guide

[Style guide](https://app.getguru.com/card/c98GRexi/Style-Guide-for-Mews-Open-API-Documentation)

#### Check during review

- [ ] The changelog and potentially a deprecation entries are in place.
  - [ ] The changelog timestamp is date of merge to develop. 
- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

MikeAdamsMews reviewed (approved):
I've edited the new entries for `CreatorProfileId` and `UpdaterProfileId`, I also added in the description for the top-level response object which I noticed was missing as well, and finally I added a brief entry to the Changelog. I verified the new property definitions from the OpenAPI YAML, but it needs someone from Dev to check over it and approve the change.

Regarding other missing instances in the API Docs, I couldn't find anything in the examples or in the OpenAPI YAML. Perhaps raise a separate PR if you find anything else @BrittaBecker?

BrittaBecker commented:
> I've edited the new entries for `CreatorProfileId` and `UpdaterProfileId`, I also added in the description for the top-level response object which I noticed was missing as well, and finally I added a brief entry to the Changelog. I verified the new property definitions from the OpenAPI YAML, but it needs someone from Dev to check over it and approve the change.
> 
> Regarding other missing instances in the API Docs, I couldn't find anything in the examples or in the OpenAPI YAML. Perhaps raise a separate PR if you find anything else @BrittaBecker?

It's returned in bills/getAll and I can see it for this endpoint also in swagger. I haven't tested & checked other endpoints.

There is also one more point to consider:
These values return the user Id, but there is no endpoint yet to get more information about the user. So these values are currently useless for partners because they still do not know who created or updated the data. 
We have the internal portfolio endpoints for enterprise profiles, but those are not available for partners. 
Depending on the rollout plan for those endpoints, maybe it's good to add a note to CreatorProfile / UpdaterProfile?
